lib/runtime/Cargo.toml

```diff
@@ -24,6 +24,14 @@ path = "../clif-backend"
 version = "0.12.0"
 optional = true
 
+# Dependencies for caching.
+[dependencies.serde]
+version = "1.0"
+# This feature is required for serde to support serializing/deserializing reference counted pointers (e.g. Rc and Arc).
+features = ["rc"]
+[dependencies.serde_derive]
+version = "1.0"
+
```

Comment:
It looks like this chunk of the Cargo.toml came from `runtime-core`, do you know if we can remove the feature from `runtime-core`?

Reply:
Unfortunately both need serde_derive as they use `#[derive(Serialize, Deserialize, ...)]`.

Reply:
Oops, the git diff shows 4 lines, I intended to point to just the top one with `features = ["rc"]`!

Reply:
It's a very minor thing, just thought it'd be better to remove it if it's not needed